spec/requests/users_controller_spec.rb

```diff
@@ -81,33 +2432,7 @@ def honeypot_magic(params)
     end
   end
 
-  describe "updating a user" do
-    before do
-      sign_in(user)
-    end
-
-    it "should be able to update a user" do
-      put "/u/#{user.username}.json", params: { name: 'test.test' }
-
-      expect(response).to be_success
-      expect(user.reload.name).to eq('test.test')
-    end
-
-    describe 'when username contains a period' do
-      before do
-        user.update!(username: 'test.test')
-      end
-
-      it "should be able to update a user" do
-        put "/u/#{user.username}.json", params: { name: 'testing123' }
-
-        expect(response).to be_success
-        expect(user.reload.name).to eq('testing123')
-      end
-    end
-  end
-
-  describe "#account_created" do
+  describe '#account_created' do
```

Comment:
Just a nitpick here but we generally want to avoid making cosmetic changes as they tend to pollute the commit history.